packages/common/src/hooks/useListData/useListData.ts

```diff
@@ -16,15 +16,15 @@ export interface ListApi<T extends ObjectWithStringKeys> {
     first: number;
     offset: number;
     sortBy: SortBy<T>;
-  }) => () => Promise<{ data: T[]; totalLength: number }>;
+  }) => () => Promise<{ nodes: T[]; totalCount: number }>;
```

Comment:
The api does `nodes/totalCount` rather than what we had `data/totalLength` - oops